services/galley/src/Galley/API/Teams.hs

```diff
@@ -250,18 +253,28 @@ createNonBindingTeamH zusr zcon (Public.NonBindingNewTeam body) = do
       (body ^. newTeamIconKey)
       NonBinding
   finishCreateTeam team owner others (Just zcon)
-  pure (team ^. teamId)
+  let tid = team ^. teamId
+  void $ E.createGlobalTeamConversation (qualifyAs lusr tid) zusr
```

Comment:
After creating the global team conversation, you have to add the creator client `c` to the group with something like this:

```haskell
  E.addMLSClients lcnv (qUntagged lusr) (Set.singleton (c, nullKeyPackageRef))
```
The same goes for the binding team.